src/test/java/com/thomashan/coup/PlayerTest.java

```diff
@@ -20,76 +20,77 @@
 class PlayerTest {
     @Test
     public void testStolenToOtherPlayer_PlayerHas3Coins() {
-        assertEquals(1, build(3).stolenToOtherPlayer().getCoins());
+        assertEquals(1, newBuilder().coins(3).build().stolenToOtherPlayer().getCoins());
     }
 
     @Test
     public void testStolenToOtherPlayer_PlayerHas2Coins() {
-        assertEquals(0, build(2).stolenToOtherPlayer().getCoins());
+        assertEquals(0, newBuilder().coins(2).build().stolenToOtherPlayer().getCoins());
     }
 
     @Test
     public void testStolenToOtherPlayer_PlayerHas1Coin() {
-        assertEquals(0, build(1).stolenToOtherPlayer().getCoins());
+        assertEquals(0, newBuilder().coins(1).build().stolenToOtherPlayer().getCoins());
     }
 
     @Test
     public void testStolenToOtherPlayer_PlayerHasNoCoins() {
-        assertThrows(IllegalArgumentException.class, () -> build(0).stolenToOtherPlayer());
+        Player player = newBuilder().coins(0).build();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found 'DU'-anomaly for variable 'player' (lines '38'-'40').](https://app.codacy.com/app/thomashan/coup-java/pullRequest?prid=4561385)